core/ui/src/components/domains/ChangeUserPasswordModal.vue

```diff
@@ -90,14 +112,51 @@ export default {
         // clear password fields
         this.newPassword = "";
         this.clearConfirmPasswordCommand++;
-
+        this.listPasswordPolicy();
         setTimeout(() => {
           this.focusPasswordField = { element: "newPassword" };
         }, 400);
       }
     },
   },
   methods: {
+    async listPasswordPolicy() {
+      this.loading.listPasswordPolicy = true;
+      this.error.listPasswordPolicy = "";
+      const taskAction = "get-password-policy";
+      const eventId = this.getUuid();
+      // register to task completion
+      this.$root.$once(
+        `${taskAction}-completed-${eventId}`,
+        this.listPasswordPolicyCompleted
+      );
+      const res = await to(
+        this.createModuleTaskForApp(this.mainProvider, {
+          action: taskAction,
+          extra: {
+            title: this.$t("action." + taskAction),
+            isNotificationHidden: true,
+            eventId,
+          },
+        })
+      );
+      const err = res[0];
+
+      if (err) {
+        console.error(`error creating task ${taskAction}`, err);
+        this.error.listPasswordPolicy = this.getErrorMessage(err);
+        return;
+      }
+    },
+    listPasswordPolicyCompleted(taskContext, taskResult) {
+      const Config = taskResult.output;
```

Comment:
Let's declare variables with leading lowercase letter

Reply:
should be good